spec/features/show_a_document_spec.rb

```diff
@@ -20,5 +20,6 @@ def then_i_see_the_document_summary
     expect(page).to have_content(@document.summary)
     expect(page).to have_content(@document.created_at.strftime("%l:%M%P on %d %B %Y"))
     expect(page).to have_content(@document.updated_at.strftime("%l:%M%P on %d %B %Y"))
+    expect(page).to have_content(I18n.t("documents.show.metadata.created_by") + ": " + @document.creator.name)
```

Comment:
I think we're not bothering with the label here - just the value is fine.

Reply:
Ah so the reason I had to include this was that without it the test would pass as the creator's name would already appear in the page in the top right hand corner.  This helps make sure we're testing the right thing.

Reply:
So I think you can use a 'within' block (which could go around all of those expectations) to scope the have_content matcher to look within the status box.

Reply:
Good idea - will implement that 👍